template/Readme.md

```diff
@@ -153,6 +153,8 @@ to unblock use the `validate: -> { false }` option.
 
  - make sure to be on update `master` to not undo other changes
  - run `bundle exec rake kennel:update_datadog`
```

Comment:
how about  change the command above (line 155) to `PROJECT=truth_service bundle exec rake kennel:update_datadog` instead? since most of the time users are only interested in a single project to test

Reply:
yeah good idea, copy-paste instructions should be safe by default :D